master/buildbot/steps/transfer.py

```diff
@@ -562,6 +562,8 @@ class JSONStringDownload(StringDownload, WorkerAPICompatMixin):
 
     name = "json_download"
 
+    renderables = ['o']
+
     def __init__(self, o, workerdest=None,
```

Comment:
Yeah, I was trying to make the minimal set of changes that would work; the parameter was already named `o` so I didn’t change it.